src/App.vue

```diff
@@ -1,88 +1,23 @@
 <template>
-    <template v-if="!loadingStatus">
+    <suspense>
```

Comment:
that was removed a while ago for causing load issues, any specific reason for adding it again?